lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1710,6 +1710,54 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     }
     gtTensor.emitError("unimplemented: dtype isn't supported.");
   }
+  if (auto eqTensor = dyn_cast<AtenEqTensorOp>(op)) {
```

Comment:
I think that `eq`, `gt`, and `lt` share a lot of code and can be refactored into one function. Maybe that can be added as a TODO.